src/main/java/org/apache/commons/ognl/ASTAdd.java

```diff
@@ -129,13 +129,6 @@ boolean isWider( NodeType type, NodeType lastType )
         {
             return false;
         }
-
-        if ( Float.class.isAssignableFrom( type.getGetterClass() )
-            && Integer.class.isAssignableFrom( lastType.getGetterClass() ) )
-        {
-            return true;
-        }
```

Comment:
Why did you remove this?

Reply:
> Why did you remove this?

Hi. @lukaszlenart 
If the code get there, always will return true


 ```
 if ( Float.class.isAssignableFrom( type.getGetterClass() )
            && Integer.class.isAssignableFrom( lastType.getGetterClass() ) )
        {
            return true;
        }

        return true;
```

Reply:
Right, but I think it can also be misleading why there is no `Float` and `Integer` check as above.